Kitodo-Docket/src/main/java/org/kitodo/docket/ExportXmlLog.java

```diff
@@ -282,7 +305,119 @@ private List<Element> prepareProperties(List<Property> properties, Namespace xml
         return preparedProperties;
     }
 
-    private String replacer(String in) {
+    private static List<Element> createMetadataElements(Namespace xmlns, DocketData docketData) {
+        List<Element> metadataElements = new ArrayList<>();
+        try {
+            HashMap<String, String> names = getNamespacesFromConfig();
+            Namespace[] namespaces = new Namespace[names.size()];
+            int index = 0;
+            for (var entries = names.entrySet().iterator(); entries.hasNext(); index++) {
+                Entry<String, String> entry = entries.next();
+                namespaces[index] = Namespace.getNamespace(entry.getKey(), entry.getValue());
+            }
+
+            prepareMetadataElements(metadataElements, false, docketData, namespaces, xmlns);
+            if (Objects.nonNull(docketData.getParent())) {
+                prepareMetadataElements(metadataElements, true, docketData.getParent(), namespaces, xmlns);
+            }
+
+        } catch (IOException | JDOMException | IllegalArgumentException e) {
+            logger.error(e.getMessage(), e);
+        }
+        return metadataElements;
+    }
+
+    private static HashMap<String, String> getNamespacesFromConfig() {
+        HashMap<String, String> nss = new HashMap<>();
+        try {
+            File file = new File(KitodoConfig.getKitodoConfigDirectory() + "kitodo_exportXml.xml");
+            if (file.exists() && file.canRead()) {
+                XMLConfiguration config = new XMLConfiguration(file);
+                config.setListDelimiter('&');
+                config.setReloadingStrategy(new FileChangedReloadingStrategy());
+
+                int count = config.getMaxIndex("namespace");
+                for (int i = 0; i <= count; i++) {
+                    String name = config.getString("namespace(" + i + ")[@name]");
+                    String value = config.getString("namespace(" + i + ")[@value]");
+                    nss.put(name, value);
+                }
+            }
+        } catch (ConfigurationException | RuntimeException e) {
+            logger.debug(e.getMessage(), e);
+            nss = new HashMap<>();
+        }
```

Comment:
This whole code block is dublicated in "new" method `getMetsFieldsFromConfig` ... maybe move this code to its own method and reuse it?